draft-ietf-quic-applicability.md

```diff
@@ -620,6 +641,31 @@ consider how the protocol will manage different QUIC versions.
 Decisions for those protocols might be informed by choices made by other
 protocols, like HTTP/3.
 
+## Source Port Selection
+
+Some UDP protocols are vulnerable to reflection attacks, where an attacker is
+able to direct traffic to a third party as a denial of service. For example,
+these source ports are associated with applications known to be vulnerable to
+reflection attacks (often due to server misconfiguration):
+
+* port 53 - DNS {{?RFC1034}}
+* port 123 - NTP {{?RFC5905}}
+* port 1900 - SSDP {{SSDP}}
+* port 5353 - mDNS {{?RFC6762}}
+* port 11211 - memcached
+
+Services might block source ports of protocols known to be vulnerable to
+reflection, to avoid the overhead of processing large numbers of packets by
+their QUIC implementations. However, this practice has negative effects on
```

Comment:
I don't think it's the port that's vulnerable to attacks; it's the underlying protocol.

Reply:
```suggestion
Services might block source ports associated with protocols known to be
vulnerable to reflection attacks, to avoid the overhead of processing large
numbers of packets. However, this practice has negative effects on
```

Reply:
split the difference between Ian's and Mark's suggestions?